### DISCUSSION TITLE
Additional language dependencies

### DISCUSSION THREAD

**Discussion by nbrownus** [codemagic-ci-cd/codemagic-docs#973] [Q&A] [answered]
I have a flutter app that depends on golang and gomobile, what is the preferred way to install this language dependency?

**Comment by himesh-cm** [comment, marked as answer]
Hi @nbrownus 

You have access to the VM, and you can follow the installation process depending upon type of software.

E.g. Installing via homebrew
https://formulae.brew.sh/formula/go

You can add it as a script step.
```
- name: install go lang
  script: |
    brew install go
    go version
```
    
You can find the list of available software pre-installed [here](https://docs.codemagic.io/specs/versions-macos-m1-xcode-13-3/).    

Let me know in case of any queries.

Thank you.
